README.md

```diff
@@ -66,6 +66,14 @@ Using [antibody](https://github.com/getantibody/antibody):
 antibody bundle ajeetdsouza/zoxide
 ```
 
+Using [ohmyzsh](https://github.com/ohmyzsh/ohmyzsh)
+
+```sh
+mkdir -p $HOME/.oh-my-zsh/custom/plugins/zoxide && curl https://raw.githubusercontent.com/ajeetdsouza/zoxide/master/zoxide.plugin.zsh -o $HOME/.oh-my-zsh/custom/plugins/zoxide/zoxide.plugin.zsh
```

Comment:
Why not clone the git repo into `custom/plugins` so that it can be updated later on with a `git pull`?